.github/workflows/ci-java.yml

```diff
@@ -61,8 +70,54 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v5
 
+      - name: Configure AWS credentials
+        id: config-aws-creds
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.NON_PROD_ACCOUNT_GHA_ROLE_ARN }}
+          role-session-name: ci-java
+          aws-region: us-east-1
+
+      - name: Mask sensitive AWS data
+        id: mask-sensitive-aws-data
+        run: |
+          caller_id="$(aws sts get-caller-identity)"
+          account_num="$(jq -r '.Account' <<<${caller_id})"
+          role_arn="$(jq -r '.Arn' <<<${caller_id})"
+          user_id="$(jq -r '.UserId' <<<${caller_id})"
+
+          echo "::add-mask::$account_num"
+          echo "::add-mask::$role_arn"
+          echo "::add-mask::$user_id"
+
+      - name: Set env vars from AWS params
+        uses: cmsgov/cdap/actions/aws-params-env-action@main
+        with:
+          params: |
+            SONAR_TOKEN=/bfd/platform/sonar/sensitive/service_account_access_key
+
       - name: 'Run Maven Build'
-        run: mvn --threads 1C --quiet --batch-mode -Dmaven.build.cache.enabled=false -Dapidocgen.skip=false -DbfdOps.skip=false verify
+        run: |
+          mvn --threads 1C \
+          --no-transfer-progress \
+          -Dmaven.build.cache.enabled=false \
+          -Dapidocgen.skip=false \
+          -Dmaven.jacoco.skip=false \
+          clean install
+        working-directory: ./apps
+
+      - name: 'Generate, Upload Build Reports'
+        run: |
+          mvn sonar:sonar \
+          -Dsonar.ci.autoconfig.disabled=true \
```

Comment:
Sonar will detect environment variables that identify this as both a CodeBuild and GitHub Actions environment, which causes it to error. This must be disabled with this codebuild via github actions strategy.